buildParts.js

```diff
@@ -0,0 +1,369 @@
+const { convertGraphQLToFQL } = require("./graphqlToFQLConverter");
+const {
+	capitalizeFirstLetter,
+	removeQuotes,
+	checkBool,
+	isNumeric
+} = require("./helper");
+const { operatorMap } = require("./constants");
+
+const functionCall = ' => ';
+const prefix = 'RE_';
+
+const getQueries = (data, queries) => {
+	if (!data)
+		return;
+
+	for (let index = 0; index < data.length; index++) {
+		const { all, any } = data[index];
+		if (all)
+			getQueries(all, queries);
+		else if (any)
+			getQueries(any, queries);
+		else {
+			const { source, target } = data[index];
+			if (source?.type?.toLowerCase() === 'fact')
```

Comment:
Store `'fact'` in constant variable like this `const FACT = 'fact'` and use variable everywhere in the code